### ISSUE TITLE
Parser enforces CRLF

### ISSUE THREAD

**Issue by emersion** [luxifer/ical#12]
Sorry for creating that many issues. I don't expect you to fix them. :P

* * *

The parser chokes on inputs which use LF instead of CRLF. While this isn't RFC-compliant (and a formatter should really use CRLF), some existing iCal files in the wild using LF already exist. It would be nice to make the parser more lenient to be able to parse them.

**Comment by luxifer** [comment]
I agree, if you can find some example in the wild of ical not properly following the RFC and add it to the fixtures I would be happy to merge it!

**Comment by emersion** [comment]
I've hit this when using the [Radicale](https://radicale.org/) CardDAV server.

**Comment by luxifer** [comment]
Got it, as I said, if you can find some example and add it to the fixtures I would be happy to make the necessary changes
